src/main/java/io/vertx/ext/mongo/MongoClient.java

```diff
@@ -172,6 +187,21 @@ MongoClient updateCollectionWithOptions(String collection, JsonObject query, Jso
 
   Future<@Nullable MongoClientUpdateResult> updateCollectionWithOptions(String collection, JsonObject query, JsonObject update, UpdateOptions options);
 
+  /**
+   * Use an aggregation pipeline to update documents in the specified collection, specifying options and return the handler with MongoClientUpdateResult result
```

Comment:
use `{@link MongoClientUpdateResult}`

Reply:
ping

Reply:
You'll have to be more specific.
I've already added the ``{@link MongoClientUpdateResult}``  (commit https://github.com/vert-x3/vertx-mongo-client/pull/254/commits/c7abe3ae5781f4107aa239c057c94ac1459293c2).
Is there something else?